### PR TITLE
feat: add flakehub entry + nix flake

### PR DESCRIPTION
**Pull request by tulilirockz**
Title! This PR should add a Flakehub entry and a Nix flake for this project :>

### CODE REVIEW

**Comment by bayou-brogrammer** [comment]
Nice was hoping to one day add a mix configuration like my os config but this is awesome. Appreciate it

**Comment by bayou-brogrammer** [comment]
Does the den shell work and build / run bluebuild? @tulilirockz we should also add earthly to the shell since that is used for integration testing

File: flake.nix [hunk 1]
```diff
@@ -0,0 +1,78 @@
+{
+  
+  description = "Accessible tools for you to create, configure and build custom images of atomic Fedora distributions.";
```
Comment (xynydev) [inline flake.nix:3]: Bluebuild build these types of tools, yes, but cli by itself is not "tools". I'd rewrite this to "A commandline tool for creating, etc..."

Comment (tulilirockz) [reply flake.nix:3]: Oh LOL sorry I just copied the description from the bluebuild website, and I didnt see that this repo had a perfectly good description here

**Comment by tulilirockz** [comment]
Everything should be fine now!

**Review by bayou-brogrammer — DISMISSED** [review]

**Comment by tulilirockz** [comment]
Sorry for modifying stuff right when you approved everything, its just that this might be quite useful for discovery

File: .envrc [hunk 2]
```diff
@@ -0,0 +1 @@
+use flake
```
Comment (gmpinder) [inline .envrc:1]: Is this a nix specific file?  Do I have to worry about any weird errors when working in the repo when I don't use nix?

File: .github/workflows/flakehub-tagged.yml
Comment (gmpinder) [inline .github/workflows/flakehub-tagged.yml:1]: Do we need to setup any kind of account with Flakehub for this to work?

**Comment by gmpinder** [comment]
Other than the couple of questions I have, I'm fine with this change. I'll rely on @bayou-brogrammer to make sure this is good since he's the local Nix user

File: .github/workflows/flakehub-tagged.yml
Comment (tulilirockz) [reply .github/workflows/flakehub-tagged.yml:1]: Nope! It just works by publishing it to flakehub and... thats it!

File: .envrc (hunk 2 above)
Comment (tulilirockz) [reply .envrc:1]: Yup, it should load the nix flake if you have nix and direnv installed on your machine, but I think there shouldnt be any issues on other machines without nix installed

**Comment by gmpinder** [comment]
Build is failing, waiting on #114 to be merged for the fix

**Review by gmpinder — APPROVED** [review]

**Pull request merged**